app/code/Magento/Customer/Controller/Address/FormPost.php

```diff
@@ -89,18 +92,21 @@ public function __construct(
             $dataProcessor,
             $dataObjectHelper,
             $resultForwardFactory,
-            $resultPageFactory
+            $resultPageFactory,
+            $customerRepository
         );
     }
 
     /**
      * Extract address from request
      *
+     * @param null $customer
      * @return \Magento\Customer\Api\Data\AddressInterface
+     * @throws \Exception
      */
-    protected function _extractAddress()
+    protected function _extractAddress($customer = null)
```

Comment:
Adding new parameters to protected methods is backward incompatible. Instead of adding parameters to protected methods, Create a new method with a new signature and deprecate the old method without changing it. https://devdocs.magento.com/guides/v2.0/contributor-guide/backward-compatible-development/